src/sentry/api/endpoints/organization_group_index_stats.py

```diff
@@ -66,7 +66,7 @@ def get(self, request: Request, organization) -> Response:
             raise ParseError(detail=str(e))
 
         expand = request.GET.getlist("expand", [])
-        collapse = request.GET.getlist("collapse", ["base"])
+        collapse = request.GET.getlist("collapse", ["base", "unhandled"])
```

Comment:
prevents double fetching unhandled since the serializer has been updated but we are not using the new endpoint yet.